### PR TITLE
I upload the magazine with html and css, task one

### PR DESCRIPTION
**Pull request by santiagoGuillenGlob**

### CODE REVIEW

File: santiao-guillen/homework-1/style.css [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+.informationLeft p{
+    width: 175px;
+    color: #ffffff;
+}
+
+.informationLeft article p:nth-child(2){
+    color: #000000;
+}
+
+hr {
```
Comment (emolinaglobant) [inline santiao-guillen/homework-1/style.css]: Move all  the Element rules to the top

File: santiao-guillen/homework-1/style.css [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    margin: 0px;
+    padding: 0px;
+}
+
+body{
+    margin: 20px 220px;
+}
+
+.backgraundImg{
+    position:relative;
```
Comment (emolinaglobant) [inline santiao-guillen/homework-1/style.css:21]: Make sure all the properties are alphabetically sorted

**Comment by emolinaglobant** [comment]
@santiagoGuillenGlob , pay attention to screen sizes that look weird on high resolution screens.

Screen Shot 2022-01-14 at 4 33 55 PM

But in general great Work!!!

File: santiao-guillen/homework-1/style.css (hunk 2 above)
Comment (santiagoGuillenGlob) [reply santiao-guillen/homework-1/style.css:21]: I have a question, what do you mean by "all the properties are alphabetically sorted", i don't understand well

**Pull request merged**